spipe/main.c

```diff
@@ -194,6 +194,9 @@ main(int argc, char * argv[])
 		goto err2;
 	}
 
+	/* proto_conn is now maintaining sas_t. */
```

Comment:
"sas_t is now owned by proto_conn".

(Just a matter of normal jargon; we usually talk about "owning" a memory allocation rather than "maintaining" it.)